gloo/test/openssl_utils.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "gloo/config.h"
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
+
+#include <string>
+
+namespace gloo {
+namespace test {
+
+extern std::string ca_key_file;
+extern std::string ca_file;
+extern std::string pkey_file;
+extern std::string csr_file;
+extern std::string cert_file;
```

Comment:
names here are inconsistent with the ones in benchmark options, any reason for this?

Reply:
addressed